index.js

```diff
@@ -4,6 +4,7 @@ const knex = require('knex');
 const knexConfig = require('./knexfile.js')
 const bcrypt = require('bcryptjs');
 const session = require('express-session');
+const secret = "secret";
```

Comment:
As we discussed before, review the concept of "scope". You can look at old JS lectures from earlier and from precourse. Lemme know if you have any questions.